### ISSUE TITLE
BBBB and AudioEngine send HW source changes to playground

### ISSUE THREAD

**Issue by hhoegelo** [nonlinear-labs-dev/C15#2547]
There seem to be redundant messages of the ribbons sent by the playcontroller via BeagleBoneBridge and via AudioEngine simultaneously. We should decide for either of the two.

**Comment by MatthiasSeeber** [comment]
I am not entirely certain, does this only apply for the Ribbons or all HW Sources? I suspect that the BBB used to propagate HW changes to the Playground, but since the inclusion of a MIDI Bridge the job has been delegated (to the MIDI I/O Event Stage inside the Synth). Therefore I would conclude that the BBB mechanism is the obsolete one that should be removed.

**Comment by KlausStrohhaecker** [comment]
Yep, I agree to dropping the UI messages via BBB(B), this would also reduce traffic on the playcontroller itself.
The disadvantage would be that the playcontroller diagnostic tools (running on BBB) can't access HW-sources anymore so maybe I'll include a option (default:OFF) that HW-sources still can be sent sent to the BBB.
